cognite/client/data_classes/assets.py

```diff
@@ -319,9 +326,10 @@ class AssetFilter(CogniteFilter):
         name (str): The name of the asset.
         parent_ids (List[int]): Return only the direct descendants of the specified assets.
         parent_external_ids (List[str]): Return only the direct descendants of the specified assets.
-        root_ids (List[Dict[str, Any]]): Only include these root assets and their descendants.
+        root_ids (List[Dict[str, Any]]): This parameter is deprecated. Use assetSubtreeIds instead. Only include these root assets and their descendants.
```

Comment:
We need to be careful with this. We'll have to regen the data classes to only take the datasets.